### DISCUSSION TITLE
(Neo)Vim plugin

### DISCUSSION THREAD

**Discussion by sashalikesplanes** [Slackadays/Clipboard#114] [Ideas]
Just installed clipboard today, and I am starting to see how it can be incorporated in my workflows. Loving it so far! Excellent work here )))

I am wondering if anyone is aware of any existing Neovim plugins that embed `cb`. I spend more than half my time inside Neovim, so this integration would be awesome.

I tried searched around for it, but questions regarding how to get Vim to play nice with system clipboard dominate search results.

If not, I might have a stab at it, anyone would be interested?

**Comment by Slackadays** [comment]
I haven't ever used neovim so I've never thought about a plugin for it. However, I do know that a lot of us (like you) do use neovim so if you can make that plugin, I think it would be very appreciated :)

**Reply by sashalikesplanes** [reply]
Thanks for the encouragement, I'll make a start and post something for feedback here

**Reply by Slackadays** [reply]
@sashalikesplanes what's the current status of the plugin?

**Reply by sashalikesplanes** [reply]
No progress currently. I am still figuring out how cb fits in my workflow
